autopilotserver/mock/server.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/lightninglabs/lightning-terminal/autopilotserverrpc"
 	"github.com/lightninglabs/lightning-terminal/rules"
 	"github.com/lightninglabs/lightning-terminal/session"
-	"github.com/lightningnetwork/lnd/lntest/node"
+	"github.com/lightningnetwork/lnd/lntest/port"
```

Comment:
Unfortunately this commit needs to be squashed with the previous one to make CI happy.